packages/react/src/analytics/integrations/GA4/types/index.ts

```diff
@@ -0,0 +1,65 @@
+import type {
+  EventData,
+  IntegrationOptions,
+  LoadIntegrationEventData,
+  SetUserEventData,
+  trackTypes,
+  TrackTypesValues,
+} from '@farfetch/blackout-analytics';
+import type { Schema } from '../../shared/types/shared.types';
+
+export type Schemas = Record<string, Schema>;
+
+export interface GA4IntegrationOptions extends IntegrationOptions {
+  measurementId: string;
```

Comment:
The properties should use the names that are defined in the `constants` module, like this:

```suggestion
[OPTION_MEASUREMENT_ID]: string;
```